reference/7/Microsoft.PowerShell.Core/Invoke-Command.md

```diff
@@ -453,106 +426,104 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id   ProcessName
+The first two commands use `Invoke-Command` to run a `Get-Process` command for the PowerShell
+process. The output of the first command includes the **PsComputerName** property, which contains
+the name of the computer on which the command ran. The output of the second command, which uses
+**HideComputerName**, does not include the **PsComputerName** column.
 
 ### Example 11: Run a script on all the computers listed in a text file
 
+This example uses the `Invoke-Command` cmdlet to run the `Sample.ps1` script on all the computers
+listed in the `Servers.txt` file. The command uses the **FilePath** parameter to specify the script
+file. This command lets you run the script on the remote computers, even if the script file is not
+accessible to the remote computers.
+
 ```powershell
 Invoke-Command -ComputerName (Get-Content Servers.txt) -FilePath C:\Scripts\Sample.ps1 -ArgumentList Process, Service
 ```
 
-This example uses the **Invoke-Command** cmdlet to run the Sample.ps1 script on all of the computers
-listed in the Servers.txt file.
-The command uses the *FilePath* parameter to specify the script file.
-This command lets you run the script on the remote computers, even if the script file is not
-accessible to the remote computers.
-
-When you submit the command, the content of the Sample.ps1 file is copied into a script block and
-the script block is run on each of the remote computers.
-This procedure is equivalent to using the *ScriptBlock* parameter to submit the contents of the
-script.
+When you submit the command, the content of the `Sample.ps1` file is copied into a script block and
+the script block is run on each of the remote computers. This procedure is equivalent to using the
+**ScriptBlock** parameter to submit the contents of the script.
 
 ### Example 12: Run a command on a remote computer by using a URI
 
+This example shows how to run a command on a remote computer that is identified by a URI. This
+particular example runs a `Set-Mailbox` command on a remote Exchange server.
+
 ```powershell
 $LiveCred = Get-Credential
-Invoke-Command -ConfigurationName Microsoft.Exchange -ConnectionUri https://ps.exchangelabs.com/PowerShell -Credential $LiveCred -Authentication Basic -ScriptBlock {Set-Mailbox Dan -DisplayName "Dan Park"}
+$parameters = @{
+  ConfigurationName = 'Microsoft.Exchange'
+  ConnectionUri = 'https://ps.exchangelabs.com/PowerShell'
+  Credential = $LiveCred
+  Authentication = 'Basic'
+  ScriptBlock = {Set-Mailbox Dan -DisplayName "Dan Park"}
+}
+Invoke-Command @parameters
 ```
 
-This example shows how to run a command on a remote computer that is identified by a URI.
-This particular example runs a **Set-Mailbox** command on a remote Exchange server.
-The backtick (\`) in the command is the PowerShell continuation character.
+The first line uses the `Get-Credential` cmdlet to store Windows Live ID credentials in the
+`$LiveCred` variable. A credentials dialog box prompts the user to enter Windows Live ID
```

Comment:
I don't believe it is a "dialog box".  There is a prompt, but even on Windows in PS 7 preview 3 there's no "dialog box".  Should validate the use of this term in both the 6 and 7 versions of this doc.